shared/actions/tracker.js

```diff
@@ -97,6 +97,11 @@ export function registerTrackerIncomingRpcs (): TrackerActionCreator {
 
 export function getProfile (username: string): TrackerActionCreator {
   return (dispatch, getState) => {
+    // If we have a pending identify no point in firing off another one
+    if (getState().tracker.pendingIdentifies[username]) {
```

Comment:
i'm a little worried about this logic.  i think we should discuss a throttle / caching logic layer that stops stuff like this from being sprinkled around. like i could imagine call fails w/o a timeout and now you can't ever id someone. etc